README.md

```diff
@@ -26,12 +26,11 @@ We are collaborating closely with the [LibreHealth Project](http://librehealth.i
 2. [Installation](#installation)
 3. [Usage](#usage)
 4. [Documentation](#documentation)
-5. [Change log](#change-log)
-6. [Testing](#testing)
-7. [Contributing](#contributing)
-8. [Security](#security)
-9. [Credits](#credits)
-10. [License](#license)
+5. [Testing](#testing)
+6. [Contributing](#contributing)
```

Comment:
Nice @chaitak-gorai , can this readme involve the tech stack we are using? at least for a start

Reply:
Sure,..
Shall I pick up the tech stacks mentioned in the about section of the repository?